### PR TITLE
Feat/portfolio improvement

### PR DESCRIPTION
**Pull request by marxeille**
## Summary by CodeRabbit

- **New Features**
  - Introduced a new modal for managing staking rewards and navigating to fiat purchases.
  - Added support for displaying network information with improved visual cues.
  - Enhanced the `AccountBoxAll` component to display detailed balance information with a pie chart.
  
- **Improvements**
  - Updated the styling and layout of various components for better user experience.
  - Simplified API interactions for retrieving validator information.
  - Enhanced error handling and user feedback mechanisms across components.

- **Bug Fixes**
  - Resolved issues related to token selection impact warnings.
  - Improved the functionality of the swap and staking features.

- **Chores**
  - Cleaned up unused imports and optimized code for better maintainability.

### CODE REVIEW

**Review by haunv3 — APPROVED** [review]
lgtm

File: packages/extension/src/pages/home/modals/modal-network.tsx [hunk 1]
```diff
@@ -77,6 +77,9 @@ export const ModalNetwork: FC<{
       ) || initPrice;
     return item;
   });
+
+  console.log("chainsInfoWithBalance", chainsInfoWithBalance);
```
Comment (toandq2009) [inline packages/extension/src/pages/home/modals/modal-network.tsx:81]: remove log

File: packages/mobile/src/screens/universal-swap/helpers.ts [hunk 2]
```diff
@@ -60,36 +60,131 @@ export const handleErrorSwap = (message: string) => {
...
+    fromCoinGeckoIds: ["usd-coin"],
+    toCoinGeckoIds: [...toCoinGeckoIds, "injective-protocol"],
+  },
+  "cosmoshub-4-Oraichain": {
+    fromCoinGeckoIds: ["cosmos"],
+    toCoinGeckoIds: [...toCoinGeckoIds],
+  },
+};
+
+// export const isAllowAlphaSmartRouter = (fromToken, toToken, isAIRoute) => {
```
Comment (toandq2009) [inline packages/mobile/src/screens/universal-swap/helpers.ts:116]: remove note comment

**Review by toandq2009 — APPROVED** [review]
lgtm

**Pull request merged**